source/_components/derivative.markdown

```diff
@@ -0,0 +1,59 @@
+---
+title: "Derivative Sensor"
+description: "Instructions on how to integrate Derivative Sensor into Home Assistant."
+ha_category:
+  - Utility
+  - Energy
+ha_release: 0.99
+ha_iot_class: Local Push
+logo: derivative.png
+ha_qa_scale: internal
+redirect_from:
```

Comment:
Please remove the `redirect_from`

Reply:
@klaasnicolaas Could tell me the reason (first pr for me)

Reply:
We used to have separate pages for an integration for each platform that it supported, in the end it was decided to merge all platform pages into 1 integration page (with a few exceptions).

We now only use the `redirect_from` to refer existing integrations to a new filename format (`sensor.test.markdown` to `test.markdown`) and therefore does not apply to new integrations.